index.js

```diff
@@ -22,6 +22,7 @@
     },
   },
   rules: {
+    '@typescript-eslint/no-extra-semi': 'off',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Typeform/eslint-config-typeform/pullRequest?prid=7367242)
```suggestion
    "@typescript-eslint/no-extra-semi": 'off',
```